srtcore/core.cpp

```diff
@@ -7927,21 +7927,23 @@ int srt::CUDT::sendCtrlAck(CPacket& ctrlpkt, int size)
     // The TSBPD thread may change the first lost sequence record (TLPKTDROP).
     // To avoid it the m_RcvBufferLock has to be acquired.
     UniqueLock bufflock(m_RcvBufferLock);
-
+    if(m_bBufferWasFull && getAvailRcvBufferSizeNoLock() > 0)
+    {
+        sendAckAgain = true;
+    }
```

Comment:
```suggestion
    const bool sendAckAgain = (m_bBufferWasFull && getAvailRcvBufferSizeNoLock() > 0);
```
and remove the `sendAckAgain` on line 7907.